install_files/ansible-base/roles/restrict_direct_access_app/templates/app_rules_v4

```diff
@@ -67,15 +62,16 @@
 -A INPUT -i lo -p all -j ACCEPT -m comment --comment "Allow lo to lo traffic all protocols"
 -A OUTPUT -o lo -p all -j ACCEPT -m comment --comment "Allow lo to lo traffic all protocols"
 
-# Drop and log all other traffic
--A INPUT -j LOGNDROP -m comment --comment "Drop all other incomming traffic"
--A OUTPUT -j LOGNDROP -m comment --comment "Drop all other outgoing traffic"
+# Don't log inbound invalid state packets related to issue #830
+-A INPUT -p tcp -m state --state INVALID -j DROP -m comment --comment "drop but don't log inbound invalid state packets"
+
+# Catch all drop rule
+-A INPUT -j LOGNDROP -m comment --comment "Drop and log all other incomming traffic"
+-A OUTPUT -j DROP -m comment --comment "Drop all other outgoing traffic"
 
 # LOGNDROP everything else
--A LOGNDROP -p tcp -m limit --limit 5/min -j LOG --log-prefix "Denied_INVALID " --log-ip-options --log-tcp-options --log-uid --log-level 4
--A LOGNDROP -p tcp -m limit --limit 5/min -j LOG --log-prefix "Denied_TCP " --log-ip-options --log-tcp-options --log-uid --log-level 4
--A LOGNDROP -p udp -m limit --limit 5/min -j LOG --log-prefix "Denied_UDP " --log-ip-options --log-uid --log-level 4
--A LOGNDROP -p icmp -m limit --limit 5/min -j LOG --log-prefix "Denied_ICMP " --log-ip-options --log-uid --log-level 4
+-A LOGNDROP -p tcp -m limit --limit 5/min -j LOG --log-ip-options --log-tcp-options --log-uid --log-level 4
+-A LOGNDROP -p udp -m limit --limit 5/min -j LOG --log-ip-options --log-uid --log-level 4
+-A LOGNDROP -p icmp -m limit --limit 5/min -j LOG --log-ip-options --log-uid --log-level 4
```

Comment:
OSSEC has a few decoders to use to parse host fw rules. When we were adding the log prefix it would group possible denies by protocol and apply the correlation rules on those by each group. Since disabled logging some of the drops the OSSEC correlation rules with the log prefix would never be met since some of the required info is not being logged.